src/app/component/profile/pages/my-edit/my-edit.component.ts

```diff
@@ -96,3 +83,13 @@ export class MyEditComponent extends WithFormCheck(PageComponent)
     console.log("Delete Submission", $event);
   }
 }
+
+MyEditComponent.LinkComponentToPageInfo({
+  category: myAccountCategory,
+  menus: {
+    actions: List<AnyMenuItem>([myAccountMenuItem, ...myAccountActions]),
+  },
+  resolvers: { [userKey]: userResolvers.show },
+}).AndMenuRoute(myEditMenuItem);
+
+export { MyEditComponent };
```

Comment:
Part of me wants to say these should be a default export so that its all one line

Reply:
🤷‍♂️ works for me but it's decently clean how it is.